cylc/uiserver/app.py

```diff
@@ -126,13 +129,12 @@ class CylcUIServer(ExtensionApp):
 
     name = 'cylc'
     app_name = 'cylc-gui'
-    default_url = "/cylc"
     load_other_extensions = True
     description = '''
     Cylc - A user interface for monitoring and controlling Cylc workflows.
     '''  # type: ignore[assignment]
     examples = '''
-        cylc gui    # start the cylc GUI
+        cylc gui [workflow]   # start the cylc GUI [at the workflow page]
     '''  # type: ignore[assignment]
```

Comment:
Maybe mention what happens if it is already running?

Also, if it is already running `cylc gui --help` doesn't work (I just get the PID message).

Also, the new `--new` option doesn't feature in the help text.